packages/dev/core/src/Meshes/meshUtils.ts

```diff
@@ -216,3 +217,25 @@ export function computeMaxExtents(
 
     return maxExtents;
 }
+
+/**
+ * Removes unreferenced vertex data from the given meshes.
+ * This is useful for cleaning up unused vertex data, such as UV sets, to reduce memory usage and stay under graphics device limits.
+ * @param meshes The array of meshes to clean up.
+ */
+export function RemoveUnreferencedVerticesData(meshes: readonly AbstractMesh[]) {

```

Comment:
Using AbstractMesh is kind of weird for this function. Removing unused UVs from an instanced mesh doesn't really make any sense since it just forwards everything to the source mesh which is a Mesh. Should this be Mesh and not AbstractMesh?

Reply:
`AbstractMesh` already has functions for mutating vertex data, so I thought it made sense to follow what seemed like the existing pattern. I'm happy to change it though if you think it makes more sense to just be `Mesh`.

Reply:
I think this function should be Mesh. I don't know why we added vertex data manipulation to AbstractMesh. It seems wrong to me, especially for InstancedMesh.